apps/prairielearn/src/tests/helperDb.ts

```diff
@@ -132,12 +123,12 @@ async function setupDatabases() {
 /**
  * @this {import('mocha').Context}
  */
```

Comment:
Either declare this with TypeScript or remove it entirely; we shouldn't use JSDoc type declarations in TypeScript files.

Reply:
Done in e0e104fa7. I didn't know you could specify this type in TS, [learn something new](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#specifying-the-type-of-this-for-functions)...